iohk/NixOps.hs

```diff
@@ -525,17 +534,30 @@ scpFromNode o c (fromNodeName -> node) from to = do
     ExitFailure code -> TIO.putStrLn $ "scp from " <> node <> " failed with " <> showT code
 
 sshForEach :: Options -> NixopsConfig -> [Text] -> IO ()
-sshForEach o c cmd = 
+sshForEach o c cmd =
   nixops o c "ssh-for-each" ("--": cmd)
 
+deployed'commit :: Options -> NixopsConfig -> NodeName -> IO ()
```

Comment:
Can you show the purpose of this command? I'd prefer not to have such hacks, but I'd like to understand the motivation as there might be an easier way.